app/policies/department_policy.rb

```diff
@@ -20,6 +20,29 @@ def initialize(user, department)
     @department = department
   end
 
+  def permitted_attributes
+    return [user_ids: [], contact_attributes: contact_attrs] if @user.superadmin?
+    return [contact_attributes: contact_attrs] if @user.department_manager?
+    []
```

Comment:
return terminates the method. So if the condition is fullfilled the array is returned and the stuff afterwards is not going to be executed.